src/lxqt-config.menu

```diff
@@ -45,12 +45,13 @@
     <Menu>
         <Name>Other settings</Name>
         <Directory>lxqt-settings-other.directory</Directory>
-        <OnlyUnallocated />
         <Include>
             <And>
                 <Category>Settings</Category>
                 <Not>
                     <Or>
+                        <Filename>kdesystemsettings.desktop</Filename>
+                        <Filename>pcmanfm-qt-desktop-pref.desktop</Filename>
```

Comment:
Very sorry, I saw the PR only now.

This section seems to be about exclusion. What about this instead of `<Filename>kdesystemsettings.desktop</Filename>`?

```xml
<Category>KDE</Category>
<Category>GNOME</Category>
```

And what about other DE categories? I think "Other settings" entries were left out to be decided later. Maybe, it's time to make a decision.

Reply:
What should be shown are  the (once removed `OnlyShowIn=XFCE;`)  three Xfwm4 settings. 
KDE and Gnome could be excluded, not sure about  others, probably yes.

Reply:
> What should be shown are the three Xfwm4 settings.

We can make exceptions for them.

I'll play with it later. It'll be good if we decide about it before the next release.